src/test/java/com/example/weather/client/mappers/ConditionsMapperTest.java

```diff
@@ -10,17 +11,29 @@
 
 class ConditionsMapperTest {
 
-    private ConditionsDto conditionsDto = makePojo(ConditionsDto.class);
+    private ConditionsDto expectedConditionsDto = makePojo(ConditionsDto.class);
+    private Conditions expectedConditions = makePojo(Conditions.class);
```

Comment:
From what I see each of those fields are used in just one method. They could be variables initialized there. Most of the tests are like this.